pkg/util/dummy_device.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2024 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/vishvananda/netlink"
+	"golang.org/x/sys/unix"
+)
+
+type DummyDeviceManager struct {
+	netlink.Handle
+}
+
+func NewDummyDeviceManager() *DummyDeviceManager {
+	return &DummyDeviceManager{netlink.Handle{}}
+}
+
+// EnsureDummyDevice ensure dummy device exist
+func (d *DummyDeviceManager) EnsureDummyDevice(devName string) (bool, error) {
+	_, err := d.LinkByName(devName)
```

Comment:
Should the error here be ignored?

Reply:
> Should the error here be ignored?

if err !=nil, we will link the devName. I have added a warning log for this error.